packages/expo-app-auth/app.plugin.js

```diff
@@ -33,6 +38,20 @@ const withAppAuth = (
     }
     return config;
   });
+  config = withInfoPlist(config, config => {
+    const infoPlist = config.modResults;
+    const existingUrlTypes = infoPlist.CFBundleURLTypes;
+    infoPlist.CFBundleURLTypes = [
+      ...(existingUrlTypes ?? []),
```

Comment:
```suggestion
      ...(existingUrlTypes || []),
```

Not supported, please test by ejecting in NCL.

Reply:
I did, looks like this support for `??` was added in node 14 which is why it's working for me locally

Reply:
Expo supports all LTS versions so node 10